apps/aetx/src/aetx.erl

```diff
@@ -174,8 +174,8 @@ fee(#aetx{ cb = CB, tx = Tx }) ->
 
 %% In case 0 is returned, the tx will not be included in the micro block
 %% candidate by the mempool.
--spec gas(Tx :: tx(), Height :: aec_blocks:height()) -> Gas :: non_neg_integer().
-gas(#aetx{type = Type, cb = CB, size = Size, tx = Tx }, Height) when
+-spec gas_limit(Tx :: tx(), Height :: aec_blocks:height()) -> Gas :: non_neg_integer().
+gas_limit(#aetx{type = Type, cb = CB, size = Size, tx = Tx }, Height) when
```

Comment:
I suggest option A below as way forward. Though only nice to have.

I understand the only semantics of this function is the contribution of the tx towards the block gas limit. Correct?
If yes, I would suggest either of the following two:
* (A) Add comment on top of this function mentioning semantics.
* (B) Rename function in order to have more explicit name (as `gas_limit` as name is used in contract txs as name for max gas specified by user, while this is derived quantity). E.g. `gas_for_micro_block_limit` though this is very debatable - and internal - so not strong opinion.

Reply:
@lucafavatella correct, the function is used for calculation of tx's contribution towards the block gas limit.